src/app/store/index.ts

```diff
@@ -1,35 +1,38 @@
-import {
-  ActionReducer,
-  ActionReducerMap,
-  createFeatureSelector,
-  createSelector,
-  MetaReducer
-} from '@ngrx/store';
-import { environment } from '../../environments/environment';
-import * as fromCountries from './countries/countries.reducer';
-import * as fromCompany from '../modules/registration/store/company.reducer';
-import * as fromCart from './cart/cart.reducer'
-import * as fromSectors from './sectors/sectors.reducer';
-import * as fromLogin from './login/login.reducer';
+import {
+  ActionReducer,
+  ActionReducerMap,
+  createFeatureSelector,
+  createSelector,
+  MetaReducer
+} from '@ngrx/store';
+import { environment } from '../../environments/environment';
+import * as fromCountries from './countries/countries.reducer';
+import * as fromCompany from '../modules/registration/store/company.reducer';
+import * as fromCart from './cart/cart.reducer'
+import * as fromSectors from './sectors/sectors.reducer';
+import * as fromLogin from './login/login.reducer';
 import * as fromCurrentCustomer from './current-customer/current-customer.reducer';
+import * as fromCompanies from '../modules/companies/store/companies.reducer';
+
+export interface State {
+[fromCountries.countriesFeatureKey]: fromCountries.State;
+[fromCompany.companyRegistrationFeatureKey]: fromCompany.State;
+[fromCart.cartFeatureKey]: fromCart.State
+[fromSectors.sectorsFeatureKey]: fromSectors.State;
+[fromLogin.loginFeatureKey]: fromLogin.State;
+[fromCurrentCustomer.currentCustomerFeatureKey]: fromCurrentCustomer.State;
[fromCompanies.companiesFeatureKey]: fromCompanies.State;
```

Comment:
No new line. This file is usually autogenerated. Did you touched it?

Reply:
Only the autogen was used. Possible I just have saved this file somewhen.